src/tests/IntegrationTest.t.sol

```diff
@@ -18,6 +18,7 @@ pragma solidity ^0.8.16;
 import "dss-interfaces/Interfaces.sol";
 
 import "../DSSTest.sol";
+import "../MCD.sol";
```

Comment:
Wondering if we should still be able to import `MCD` and `MCDUser` from `DssTest` itself, so whenever we use it we just need to import that file.

Reply:
Maybe just makes sense this new way, honestly I'm personally not sure what is better.

Reply:
Yeah I would be okay with leaving it in the `DssTest` import.